.github/workflows/CI.yml

```
@@ -0,0 +1,55 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.4]
+        julia-arch: [x64]
+        os: [ubuntu-latest, macOS-latest]
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v1.0.0
+
+      - name: Set up Julia
+        uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+
+      - name: Add package versions
+        run: julia --project=temp_for_test -e 'using Pkg; pkg"dev .; add https://github.com/JuliaPlots/MakieRecipes.jl; add CairoMakie#master"'
+
+      - name: Run tests
+#        uses: julia-actions/julia-runtest@master
+        run: julia --project=temp_for_test -e 'using Pkg; pkg"test"'
```

This will not work until the package gets registered anyway, so I think we can just leave it at the version before 7e2ca27d62cf872f1edb14cb6bcb7202cac724ee ie with julia-actions/julia-runtest@master.

---

Couldn't we do 
```suggestion
        run: julia --project=temp_for_test -e 'using Pkg; pkg"test RecipesPipeline"'
```
?

---

Nice trick. I think it now fails for unrelated reasons.

---

Yeah, looks like CairoMakie doesn't want to precompile for some bizarre reason.  It works perfectly fine everywhere else :D

---

The CI is doing its job by showing the problems :)